.gitignore

```diff
@@ -7,3 +7,4 @@ build-storybook
 .DS_Store
 packages/ui/.env
 packages/ui/src/services/i18n/utils/csv
+yarn.lock
```

Comment:
`yarn.lock` shouldn't be in git ignore, plus add newline on the end of the file

Reply:
done